src/renderer/services/chain/fees/swap.ts

```diff
@@ -0,0 +1,62 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { Fees } from '@xchainjs/xchain-client'
+import { Chain, Asset, BNBChain, THORChain, BTCChain, ETHChain, baseAmount, BaseAmount } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/lib/function'
+import * as Rx from 'rxjs'
+
+import { getChainAsset } from '../../../helpers/chainHelper'
+import { LiveData, liveData } from '../../../helpers/rx/liveData'
+import * as BNB from '../../binance'
+import * as BTC from '../../bitcoin'
+import * as THOR from '../../thorchain'
+import { FeesLD, SwapFeesLDS } from '../types'
+
+const reloadSwapFees = () => {
+  BNB.reloadFees()
+  BTC.reloadFees()
+  THOR.reloadFees()
+}
+
+/**
+ * @todo rethink about using common fees at all views
+ * and move view-related calculations directly to the views
+ * instead of creating fee-services for every situations
+ */
```

Comment:
As discussed yesterday, we won't have common fees in case of BTC. Since it needs to know a memo to calculate fees and a memo will always be different. Also we don't want have much calculations in a view, which makes hard to test ...